src/test/java/io/stargate/sgv2/jsonapi/testresource/DseTestResource.java

```diff
@@ -18,7 +18,7 @@ public DseTestResource() {
     }
 
     if (null == System.getProperty("testing.containers.cluster-version")) {
-      System.setProperty("testing.containers.cluster-version", "6.8");
+      System.setProperty("testing.containers.cluster-version", "next");
```

Comment:
I think that instead of doing this, we need to use `testing.containers.cluster-persistence`, set that to `dse-next`.